unstructured/file_utils/filetype.py

```diff
@@ -324,7 +110,8 @@ def detect_filetype(
         else:
             return FileType.XML
 
-    elif mime_type in TXT_MIME_TYPES or mime_type.startswith("text"):
+    # -- ref: https://www.rfc-editor.org/rfc/rfc822 --
```

Comment:
What in the link is important? It's a somewhat long doc

Reply:
Dunno. The comment was already in there. That RFC is the spec for internet email. I'll cut that line.